### PR TITLE
docs: add @angular/http to list of dependencies in universal guide

### PR DESCRIPTION
**Pull request by deisterhold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/universal.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ If your server handles HTTP requests, you'll have to add your own security plumb
 Install `@angular/platform-server` into your project. Use the same version as the other `@angular` packages in your project. You also need `ts-loader`, `webpack-cli` for your webpack build and `@nguniversal/module-map-ngfactory-loader` to handle lazy-loading in the context of a server-render.
 
 <code-example language="sh" class="code-shell">
-$ npm install --save @angular/platform-server @nguniversal/module-map-ngfactory-loader ts-loader webpack-cli
+$ npm install --save @angular/http @angular/platform-server @nguniversal/module-map-ngfactory-loader ts-loader webpack-cli
```
Comment (wKoza) [inline aio/content/guide/universal.md]: Why would you install a deprecated package in your app ?

**Comment by gkalpak** [comment]
I think it is because the universal example depends on it. The right fix, of course, is not to depend on `@angular/http`, but on `@angular/common/http`.
I think this will be fixed by #28296.

**Comment by brandonroberts** [comment]
@gkalpak some context here https://github.com/angular/angular/pull/28408. This causes an issue with the latest version of the Angular CLI with those still using Angular version 7 because `@angular/platform-server` still depends on it.

**Comment by gkalpak** [comment]
Actually, I'm wrong 😱 `platform-server` apparently still [depends on @angular/http][1]. The `package.json` used for the `universal` example has [`@angular/http` as a dependency][2], but the default cli-generated `package.json` does not (as mentioned in #28218), thus following the `universal` guide instructions leaves the user in a broken state.

So, I think this is probably a correct (temporary) fix (until `platform-server` does not depend on `@angular/http`). 

[1]: https://github.com/angular/angular/blob/e18a52e24a834f8795f8ed07b446f2a21ef28708/packages/platform-server/src/http.ts#L12
[2]: https://github.com/angular/angular/blob/e18a52e24a834f8795f8ed07b446f2a21ef28708/aio/tools/examples/shared/boilerplate/universal/package.json#L24

**Comment by gkalpak** [comment]
@brandonroberts, yeah, I realized that after my first comment :grin:
Why are you saying "Angular version 7"? Isn't this affecting master as well? AFAICT, `platform-server` is still depending on `@angular/http`.

Shouldn't the PR target both master and 7.2.x?

**Comment by brandonroberts** [comment]
Closing as the universal guide has been rewritten.

**Pull request closed without merging**